qa-tests-backend/src/test/groovy/GroupsTest.groovy

```diff
@@ -161,17 +233,11 @@ class GroupsTest extends BaseSpecification {
 
         where:
         "Data inputs are"
-        authProviderName | key | id | value |
-                expectGroup | expectGroups
-        "groups-test-provider-1" | null  | GROUPS_WITH_IDS["QAGroupTest-Group1"].props.getId() | null  |
-                "Group1"    | ["Group1", "Group2"]
-        null                     | "foo" | "some-id"                                           | "bar" |
-                null        | ["Group2", "Group3"]
-        "groups-test-provider-1" | "foo" | GROUPS_WITH_IDS["QAGroupTest-Group2"].props.getId() | "bar" |
-                "Group2"    | ["Group2"]
-        "groups-test-provider-2" | null  | "some-id"                                           | null  |
-                null        | ["Group3"]
-        "groups-test-provider-2" | "foo" | GROUPS_WITH_IDS["QAGroupTest-Group3"].props.getId() | "bar" |
-                "Group3"    | ["Group3"]
+        authProviderName | key | id | value | expectGroup | expectGroups
+        "groups-test-provider-1" | null  | GROUPS_WITH_IDS["QAGroupTest-Group1"].props.getId() | null  | "Group1"    | ["Group1", "Group2"]
+        null                     | "foo" | "some-id"                                           | "bar" | null        | ["Group2", "Group3"]
+        "groups-test-provider-1" | "foo" | GROUPS_WITH_IDS["QAGroupTest-Group2"].props.getId() | "bar" | "Group2"    | ["Group2"]
+        "groups-test-provider-2" | null  | "some-id"                                           | null  | null        | ["Group3"]
+        "groups-test-provider-2" | "foo" | GROUPS_WITH_IDS["QAGroupTest-Group3"].props.getId() | "bar" | "Group3"    | ["Group3"]
```

Comment:
Guess we make it ugly again with the linter complaining
![madge](https://user-images.githubusercontent.com/89904305/232664663-ed655098-c499-4181-9056-f8e13f6614e2.gif)

Reply:
I put `SupressWarnings('RuleLength')` instead. Splitting test parameter lines makes them pretty much unreadable IMHO